packages/onboarding/src/cart/index.tsx

```diff
@@ -206,6 +207,9 @@ export const createSiteWithCart = async (
 					? { segmentation_survey_answers_anon_id: segmentationSurveyAnswersAnonId }
 					: {} ),
 				...( siteGoals && { site_goals: siteGoals } ),
+				...( shouldEnableWriteGoalFeatures && {
+					should_enable_write_goal_features: true,
+				} ),
```

Comment:
Curious if we can use the site_goals on the WPCOM side to enable features instead of passing an additional flag here?

Since goals are saved as an option on the site, could we read those after site creation to determine which features to enable?

Reply:
Ah I see, I had some confusion from the naming of `should_enable_write_goal_features`. It sounded like this flag was only intended for the write goal, but perhaps it could be used to control feature enablement for any goal.